Affiliate program.md

```diff
@@ -0,0 +1,25 @@
+Below is what I managed to prepare for a few hours with the ChatGPT and my previous research.
+
+The topic needs further discussion.
+
+**Mechanism for creating value:**
+- Companies (AWS, Notion, etc) pay affiliate commission to partners for attracting clients. The commission is from 5% to 70% of the purchase amount.
+- Ubiquity can become a partner of such companies and transfer affiliate commission to bounty hunters.
+- For this bounty, hunters must pay with tokens.
+ The token can be UBQ or a new one - UXP (Ubiquity Experience), KARMA (karma of the developer), etc.
```

Comment:
I am a bit hesitant to introduce new tokens but open minded to it. Does it not make sense to use any of our existing tokens?

As a sidenote, I was thinking it might make sense to rebrand uAD into uUSD

Reply:
My personal attitude is also not to create new tokens, but to develop an ecosystem and enhance utility of the existing tokens.

At the same time, what I see on the market is a constant change where new tokens can receive more attention than old ones simply because they are new.

uUSD sounds really great!

Reply:
> At the same time, what I see on the market is a constant change where new tokens can receive more attention than old ones simply because they are new.

technically we haven't done a massive marketing campaign, we don't even have liquidity lol, so UBQ for the purpose of the market is a new token

agree with @pavlovcik  on no new tokens, splitting liquidity is annoying, however there are projects running double token setups (Dopex is the first one coming to mind rn)